.prettierrc.json

```diff
@@ -1,4 +1,5 @@
 {
   "endOfLine": "auto",
-  "htmlWhitespaceSensitivity": "ignore"
+  "htmlWhitespaceSensitivity": "ignore",
+  "trailingComma": "all"
```

Comment:
`package.json` contains an update to Prettier 3 a few lines below, in which the default value of `trailingComma` is `all`. Why then explicitly set it to `all` here, incurring a customization that has to be maintained that does not differ from the default?